src/components/SelectionList/BaseSelectionList.tsx

```diff
@@ -225,6 +222,17 @@ function BaseSelectionList<TItem extends ListItem>(
         [flattenedSections.allOptions],
     );
 
+    // If `initiallyFocusedOptionKey` is not passed, we fall back to `-1`, to avoid showing the highlight on the first member
+    const [focusedIndex, setFocusedIndex] = useArrowKeyFocusManager({
+        initialFocusedIndex: flattenedSections.allOptions.findIndex((option) => option.keyForList === initiallyFocusedOptionKey),
+        maxIndex: flattenedSections.allOptions.length - 1,
+        isActive: true,
+        onFocusedIndexChange: (index: number) => {
+            scrollToIndex(index, true);
+        },
+        isFocused,
+    });
```

Comment:
When we are using the `useArrowKeyFocusManager` here, we should have passed the `disabledIndexes` props, which caused this issue: https://github.com/Expensify/App/issues/40583